### ISSUE TITLE
Update repository with yarn 2

### ISSUE THREAD

**Issue by bertearazvan** [DHI/react-components#253]
Hi all,

We had a few issues in the recent past in regards to the pipeline runs, and how much does a pipeline take to install dependencies, build the app and publish it with azure web apps. 

Initially, the pipeline took around **9-11 minutes** to finish, while in #251 where yarn 2 was implemented, it took around **~5 minutes** (and it can be taken even further). I and @rihr have started using _yarn 2_ for a few months now in our Data 'n Analytics apps, and everything works well, also in production. We find that it behaves well, without any bugs, more performant and a slightly prettier CLI.

In theory, it does not change anything on your end. Everything would be set up already and it will be the same as the good old `yarn install` when working with the `react-components` package. You can read more about the changes here https://medium.com/swlh/getting-started-with-yarn-2-and-typescript-43321a3acdee

So we were wondering, are you good with making this change? 

CC: @goneale @nfour @jharper93 @FranzThomsen1089 @rosudhi

**Comment by nfour** [comment]
I'd recommend replacing `yarn` with `pnpm`. You will get the same speed up without the chaos of yarn 2's eccentric module implementation. Has been working very well for my projects for the last 6month. Has also demonstrated 17min -> 2min compile time improvements in CI.

**Comment by bertearazvan** [comment]
We looked over the `pnpm` vs `yarn` and it applies an interesting concept, which I find has great potential in the future. As of now, I believe that yarn has wider community support through articles, usage, issues etc. and it is not as young as `pnpm`. 

Since most of the developers within our company develop using either `yarn` or `npm`, I am proposing to keep using `yarn`, and once `pnpm` is more stable and has a bigger ecosystem to re-evaluate it.

I tried implementing it on #251 but I can see that some package under the hood doesn't know where to look for its plugin (`@svgr`), so I changed it back to using `yarn 2`.

**Comment by bertearazvan** [comment]
We looked over the `pnpm` vs `yarn` and it applies an interesting concept, which I find has great potential in the future. As of now, I believe that yarn has wider community support through articles, usage, issues etc. and it is not as young as `pnpm`. 

Since most of the developers within our company develop using either `yarn` or `npm`, I am proposing to keep using `yarn`, and once `pnpm` is more stable and has a bigger ecosystem to re-evaluate it.

I tried implementing it on #251 but I can see that some package under the hood doesn't know where to look for its plugin (`@svgr`), so I changed it back to using `yarn 2`.
